package.json

```diff
@@ -9,7 +9,7 @@
   "gxDependencies": [
     {
       "author": "whyrusleeping",
-      "hash": "Qme1boxspcQWR8FBzMxeppqug2fYgYc15diNWmqgDVnvn2",
+      "hash": "QmaPHkZLbQQbvcyavn8q1GFHg6o6yeceyHFSJ3Pjf3p3TQ",
```

Comment:
I verified `QmaPHkZLbQQbvcyavn8q1GFHg6o6yeceyHFSJ3Pjf3p3TQ` is correctly derived from https://github.com/golang/crypto/commits/b2fa06b6af4b7c9bfeb8569ab7b17f04550717bf. Diff: https://ipfs.io/ipfs/QmWJe7zvzbKKoZ6rbmTxgbczn9FeS7zovrcawMkNYuqyuf